### ISSUE TITLE
Bug? Can't exit building

### ISSUE THREAD

**Issue by ndrwstn** [MegaMek/megamek#1029]
### Environment
Megamek 0.44.0
MacOS 10.13.5
Java 8 Update 171?

### Description
Warhammer skidded into a building Hex2119. Pathfinding says I must go through another building to exit, but I should be able to stand and exit directly north (the same way I came in?)

### Files
Save attached.
[cantleavebuilding.sav.gz](https://github.com/MegaMek/megamek/files/2177227/cantleavebuilding.sav.gz)

**Comment by wildj79** [comment]
So, it looks like you should be able to move to the hex to the north. The steps you need to take to move there are not intuitive, though. 

1. You need to "Get Up", by clicking on the "Get Up" button.
2. Then, shift + click on the hex to the North (this sets the hex-side you are facing one to left)
3. Click on the "More..." button until you can see the "Climb Mode" button
4. Click on the "Climb Mode" button until the mech has the words "Go Thru" overlaid the top of it
5. Now, click on the hex to the North.

These steps will get you into that hex, that's assuming you pass the check to stand.

**Comment by wildj79** [comment]
@inqistor It would be very helpful if you could provide a saved game that shows this behavior.

**Comment by inqistor** [comment]
@wildj79 yeah, but it was few days ago, and I haven't saved that battle. I thought this might be some vague rule, where reinforced buildings are treated like dense forest, or something like that. I'll try to replicate this. 
I recall, I had only light/medium tanks,  in two separate buildings, and infantry could leave them without problem.

**Comment by HammerGS** [comment]
Closing due to age. Please test with the latest milestone, 0.50.11, to see if the issue still persists. If it does, please re-open.
